script.js

```diff
@@ -0,0 +1,689 @@
+
+let test = '`';
+let test2 = '/';
+
+
+document.body.insertAdjacentHTML("afterbegin", `<div class="wrapper">
+<div class="textArea">
+  <textarea name="textarea" id="name" class="textarea" cols="30" rows="10" autofocus  placeholder="Write something here"></textarea>
+</div>
+<div class="virtualKeyboard">
+  <div class="keyboard_row">
+    <div class="key" id="Backquote">
+      <span class="Back_quote on">
+        <span class="case off">Ё</span>
+        <span class="case on">ё</span>
```

Comment:
1) I can only guess that `on` and `off` relates to `Shift` button. But I MUST understand it from your code. Rename this classes.
2) Why do you need to have `on` in two places? The second is for the language? Then why they are named the same?